webdriver/tests/support/helpers.py

```diff
@@ -82,6 +82,40 @@ def _restore_windows(session):
     _switch_to_top_level_browsing_context(session)
 
 
+async def cleanup_bidi_session(session):
```

Comment:
Now that we have a single depth only would it make sense to actually forward the `send_blocking_command` reference from the bidi session fixture?

Reply:
`send_blocking_command` depends on `bidi_session`, so it will be a recursive dependency. Theoretically, instead of fixture, I can make it by calling `send_blocking_command(self)` in [`bidi_session` fixture](https://github.com/web-platform-tests/wpt/blob/4bacd50eeb7318a41189ddd6f9a52bb7da84aae0/webdriver/tests/support/fixtures.py#L230), but I don't find this solution an elegant.

Reply:
Ok, maybe we can find a better solution in the future. Fine for me for now.